plugin.yaml

```diff
@@ -1,21 +1,40 @@
 plugins:
   gcp_plugin:
     executor: central_deployment_agent
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/update-missing-resource-id.zip
+    source: cloudify-gcp-plugin
     package_name: cloudify-gcp-plugin
-    package_version: '1.4.1'
+    package_version: '1.4.2'
+
+dsl_definitions:
+  use_external_resource_desc: &use_external_resource_desc >
+    Indicate whether the resource exists or if Cloudify should create the resource,
+    true if you are bringing an existing resource, false if you want cloudify to create it.
+
+  resource_id_desc: &resource_id_desc >
+    The AWS resource ID of the external resource, if
+    use_external_resource is true. Otherwise it is an empty string.
```

Comment:
@GhalebKhaled typo here please change/remove the AWS from the desc

Reply:
good one